Scripts/build

```diff
@@ -24,9 +24,9 @@ res = Net::HTTP.get_response(endpoint)
 if res.is_a?(Net::HTTPSuccess)
   puts "Using Storefront API version: #{storefront_api_version}"
 
-  output = ERB.new(File.read("version.swift.erb")).result()
+  output = ERB.new(File.read("./Scripts/version.swift.erb")).result()
```

Comment:
So we are assuming that `CWD=mobile-buy-ios/.` everytime the build script is run. However the script itself is in `mobile-buy-ios/Scripts`. 

Have we considered 
- the impact of this change on our current travis job? 

Remember to run it locally

Reply:
The previous implementation assumed that this was being run from the scripts directory, and also made assumptions about the computer it was being run on.